applications/data_analysis/climatology.ipynb

```diff
@@ -0,0 +1,3551 @@
+{
```

Comment:
<p>I might be missing something - should this section be the std of the monthly means or the std?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>